docker/Dockerfile

```diff
@@ -0,0 +1,117 @@
+# Build all the bins
+FROM golang:buster AS builder
+
+ENV APP     /go/src/Dominator
+ENV COMMIT  ee0024f8ccc1ce647137e1c75dc68f40e19c0ed2
```

Comment:
Do you have any plan to start making release tags in dominator? We could start pushing versioned images to docker hub then.